sdk/keyvault/keyvault-keys/samples/v4/typescript/README.md

```diff
@@ -12,29 +12,33 @@ urlFragment: keyvault-keys-typescript
 
 These sample programs show how to use the TypeScript client libraries for Azure Key Vault Keys in some common scenarios.
 
-| **File Name**                   | **Description**                                                  |
-| ------------------------------- | ---------------------------------------------------------------- |
-| [cryptography.ts][cryptography] | uses a key to sign/verify, encrypt/decrypt, and wrap/unwrap data |
-| [helloWorld.ts][helloworld]     | creates, reads, lists, and deletes keys                          |
-| [purgeAllKeys.ts][purgeallkeys] | purges all the keys of a Key Vault (useful for repeated tests)   |
+| **File Name**                   | **Description**                                                                 |
+| ------------------------------- | ------------------------------------------------------------------------------- |
+| [cryptography.ts][cryptography] | Demonstrates using a key to sign/verify, encrypt/decrypt, and wrap/unwrap data. |
```

Comment:
Nit: these all ended up pretty same-y, and the older format is more concise and direct IMO. "uses", "creates", and "purges" are all strong active verbs, and "demonstrates" isn't as much. We found that when we tried to make all the sample summaries into full sentences rather than fragments, we always ended up with something like "This sample demonstrates" or "This sample shows" at the beginning, so typically I prefer the more concise form.